### PR TITLE
Show RPC block number on chain page

### PR DESCRIPTION
**Pull request by FrederikBolding**
Show RPC block number and latency on the chain details page.

Closes https://github.com/FrederikBolding/chainlist/issues/43

### CODE REVIEW

File: src/components/RpcTable.tsx [hunk 1]
```diff
@@ -10,34 +11,105 @@ import {
...
+
+export const RpcTable = ({
+  chainId,
+  rpcs,
+  handleRpcClick,
+}: Pick<ChainData, "chainId"> & {
+  rpcs: ChainData["rpc"];
+  handleRpcClick: (rpc: string) => void;
+}) => {
   const { isConnected, handleConnect } = useContext(Web3Context);
```
Comment (LordGoda) [inline src/components/RpcTable.tsx:45]: Metamask

File: src/components/RpcTable.tsx [hunk 2]
```diff
@@ -10,34 +11,105 @@ import {
...
+  const [rpcResults, setRpcResults] = useState<RpcResult[] | null>(null);
+
+  useEffect(() => {
```
Comment (LordGoda) [inline src/components/RpcTable.tsx:48]: Chainlist

**Review by LordGoda — APPROVED** [review]

File: src/components/RpcTable.tsx (hunk 1 above)
Comment (Oliv23ava) [reply src/components/RpcTable.tsx:45]: oliver.2023.avalos.cb.id
0x2cb8170c91fDd19FDB3dC8f1e687baa2B6DAC337
CoinBase.
https://github.com/FrederikBolding/chainlist/pull/187#discussion_r1875034613
> const { isConnected, handleConnect } = useContext(Web3Context); + const [rpcResults, setRpcResults] = useState<RpcResult[] | null>(null); + + useEffect(() => {0x2cb8170c91fDd19FDB3dC8f1e687baa2B6DAC337}

File: src/components/RpcTable.tsx (hunk 2 above)
Comment (Oliv23ava) [reply src/components/RpcTable.tsx:48]: 0x2cb8170c91fDd19FDB3dC8f1e687baa2B6DAC337
CoinBase 
Oliver.2023.avalos.cb.id
Metamask.0xf931a800B8B96453674d78A75D83593432Ae26d4.wallet
https://github.com/FrederikBolding/chainlist/pull/187#discussion_r1875034613
 > const { isConnected, handleConnect } = useContext(Web3Context);
+ const [rpcResults, setRpcResults] = useState<RpcResult[] | null>(null);
+
+ useEffect(() => {0x2cb8170c91fDd19FDB3dC8f1e687baa2B6DAC337}
Oliver Joaquín Madera Avalos 
Oliver23Avalos

**Pull request merged**